Frontend/src/app/views/dashboard/project-details/project-details.component.ts

```diff
@@ -120,6 +124,19 @@ export class ProjectDetailsComponent {
 
     this.loadingProject = true;
 
+    this.projectService.updateProjectDetails(
+      this.updateProjectForm.get("details").value,
+      this.projectId
+    ).subscribe((response: GenericResponseDTO) => {
+      if(response.success == true){
+        this.details.description = this.updateProjectForm.get("details").value;
+        this.pageMode = 'view';
+        this.toastrService.success("The project has been saved successfully", "Project Saved");
+      }else{
+        this.toastrService.danger("There was an error updating the project", "Error");
+      }
+    })
+
```

Comment:
Because the other API call is being made separately there are 2 toastr notifications

Reply:
fixed